build/defines.mak

```diff
@@ -21,6 +21,7 @@ WHEEL := $(OUTPUT_DIR)/dist/$(DRIVER)-$(VERSION)-py2.py3-none-any.whl
 SDIST := $(OUTPUT_DIR)/dist/$(DRIVER)-$(VERSION).tar.gz
 
 MKDIRECTORIES += \
+                 $(DRIVER_DOCS_DIR) \
```

Comment:
It exercises the documentation code generation and templates. I lean towards leaving it in.